### PR TITLE
[CELEBORN-355][BUILD] Create shaded module for Celeborn common

### PR DESCRIPTION
**Pull request by kerwin-zk**
### What changes were proposed in this pull request?
Create shaded module for Celeborn common

### CODE REVIEW

**Review by FMX — APPROVED** [review]
LGTM. Thanks. But I think there should be another PR to check weather client-shaded jars are necessary.

**Comment by pan3793** [comment]
The PR description does not explain "Why are the changes needed?", so why do we need it?

File: pom.xml [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
   <modules>
     <module>common</module>
+    <module>common-shade</module>
```
Comment (pan3793) [inline pom.xml:35]: shade => shaded

**Comment by FMX** [comment]
> The PR description does not explain "Why are the changes needed?", so why do we need it?

The Celeborn is integrating with Gluten. I believe that the details should be told by @kerwin-zk .

**Comment by waitinfuture** [comment]
@kerwin-zk please add more details in description

**Comment by FMX** [comment]
This PR is revoked due to further discussion.

**Comment by waitinfuture** [comment]
We should be more careful not to merge to-revert PRs

**Pull request merged**